pkg/operator/utils/gcp/utils_test.go

```diff
@@ -107,6 +107,14 @@ func perms(low, hi int) []string {
 	return result
 }
 
+func createStringSet(stringArray []string) sets.String {
+	stringSet := sets.NewString()
+	for _, str := range stringArray {
+		stringSet.Insert(str)
+	}
+	return stringSet
```

Comment:
I think you can pass in your initial list of strings into the NewString() call.
```suggestion
        return sets.NewString(stringArray...)
```